modules/seqexec/server/src/main/scala/seqexec/server/gpi/GPI.scala

```diff
@@ -4,28 +4,41 @@
 package seqexec.server.gpi
 
 import cats.data.EitherT
-import cats.effect.{Effect, IO, Sync}
+import cats.effect.Effect
+import cats.effect.IO
+import cats.effect.Sync
 import cats.implicits._
 import cats.data.Reader
 import edu.gemini.spModel.config2.Config
 import edu.gemini.spModel.gemini.gpi.Gpi._
 import edu.gemini.spModel.seqcomp.SeqConfigNames._
-import java.lang.{Boolean => JBoolean, Double => JDouble, Integer => JInt}
-
+import java.lang.{ Boolean => JBoolean }
+import java.lang.{ Double => JDouble }
+import java.lang.{ Integer => JInt }
 import fs2.Stream
 import seqexec.model.dhs.ImageFileId
-import seqexec.model.enum.{Instrument, Resource}
+import seqexec.model.enum.Instrument
+import seqexec.model.enum.Resource
 import seqexec.server.ConfigUtilOps._
 import seqexec.server._
 import seqexec.server.gpi.GPIController._
-import seqexec.server.keywords.{GDSClient, GDSInstrument, KeywordsClient}
-
+import seqexec.server.keywords.GDSClient
+import seqexec.server.keywords.GDSInstrument
+import seqexec.server.keywords.KeywordsClient
 import scala.concurrent.duration._
-import squants.time.{Milliseconds, Seconds, Time}
+import squants.time.Milliseconds
+import squants.time.Seconds
+import squants.time.Time
 
 final case class GPI[F[_]: Effect](controller: GPIController[F])
     extends InstrumentSystem[F]
     with GDSInstrument {
+  // Taken from the gpi isd
+  val readoutOverhead: Time  = Seconds(4)
+  val writeOverhead: Time    = Seconds(2)
+  val perCoaddOverhead: Time = Seconds(2.7)
+  val extraObsTimeout: Time  = Seconds(30)
```

Comment:
This is arbitrary. We want to have a timeout of the calculated time plus some extra as a timeout. We don't want it to be so short that it timeouts when the obs gets a bit late, nor too long that we have to wait to much for errors to be detected. This value worked fine in my tests